examples/tests/invalid/statements_if.w

```diff
@@ -0,0 +1,5 @@
+let n = 0;
+--n;
```

Comment:
Maybe add the compiler diagnostics as comments like in other tests (eventually we will check them)